test/IRGen/protocol_resilience.sil

```diff
@@ -296,7 +296,13 @@ bb0(%0 : $*ResilientConformingType):
 
 // CHECK-LABEL: define{{( protected)?}} i8** @_T019protocol_resilience23ResilientConformingTypeV010resilient_A005OtherC8ProtocolAAWa()
 // CHECK-NEXT:  entry:
-// CHECK-NEXT:    [[WTABLE:%.*]] = call i8** @swift_rt_swift_getGenericWitnessTable(%swift.generic_witness_table_cache* @_T019protocol_resilience23ResilientConformingTypeV010resilient_A005OtherC8ProtocolAAWG, %swift.type* null, i8** null)
+// CHECK-NEXT:    %conditional.tables = alloca %swift.witness_table_slice, align 8
```

Comment:
See above, this is unnecessary code size bloat in the unconditional conformance case

Reply:
I guess we can skip storing the `i8***` but the 0 is important, I think, as above.